virtual_ecosystem/core/core_components.py

```diff
@@ -151,6 +153,10 @@ def __post_init__(self, config: Config) -> None:
 
         self.n_updates = int((self.end_time - self.start_time) / self.update_interval)
 
+        # Calculate the number of updates in one year
+        seconds_per_year = np.timedelta64(31536000, "s")
+        self.updates_per_year = seconds_per_year / self.update_interval
+
```

Comment:
This is a bit of a hack, but I don't think we should try anything more complex while our time handling is so simplistic.

```suggestion
        # Calculate the number of updates in one year
        # TODO - this is not calendar aware - variable length months and leap years.
        seconds_per_year = np.timedelta64(31536000, "s")
        self.updates_per_year = seconds_per_year / self.update_interval

```